src/applications/vaos/new-appointment/components/TypeOfCarePage/TypeOfCareAlert.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import recordEvent from 'platform/monitoring/record-event';
+
+export default function TypeOfCareAlert() {
+  const headline = 'Not seeing the type of care you need?';
+  return (
+    <AlertBox
+      status="info"
+      headline={headline}
+      className="vads-u-margin-top--3 vads-u-margin-bottom--7"
+      content={
+        <p>
+          You'll need to call your VA health facility to schedule an
```

Comment:
The "You'll" here should use a right single quote `’`.

Reply:
Ah, yes that bugger. Fixed